data/json/items/magazines.json

```diff
@@ -71,5 +71,218 @@
         "alternatives" : [
             [ "9mm", [ "glockmag" ] ]
         ]
+    },{
+        "type" : "MAGAZINE",
+        "id" : "m1918mag",
+        "//" : "Compatable magazines made for reproductions exist, and the standard-capacity mags are fairly cheap compared to variants.",
+        "price" : 3600,
+        "name" : "BAR 20-round magazine",
+        "symbol" : "#",
+        "color" : "ltgray",
+        "description" : "A standard capacity magazine for use with the Browning Automatic Rifle, holds 20 rounds.",
+        "material" : "steel",
+        "volume" : 2,
+        "weight" : 198,
+        "to_hit" : -2,
+        "ammo_type" : "3006",
+        "capacity" : 20,
+        "reliability" : 9,
+        "reload_time" : 30
+    },{
+        "type" : "MAGAZINE",
+        "id" : "m1918mag_30",
+        "price" : 8500,
+        "name" : "BAR 30-round magazine",
+        "symbol" : "#",
+        "color" : "ltgray",
+        "description" : "An aftermarket magazine for use with the Browning Automatic Rifle, holds 30 rounds.",
+        "material" : "steel",
+        "volume" : 2,
+        "weight" : 297,
+        "to_hit" : -2,
+        "ammo_type" : "3006",
+        "capacity" : 30,
+        "reliability" : 8,
```

Comment:
Ah, possibly. It is 1 less than the 20-round, but bumping it down to 7 might be acceptable.